Add Maze Escaper!

QuizzityMC opened the pull request:
# Your checklist for this pull request

## Maze Escaper!

**By QuizzityMC!**

## About your game

**On a dark and lonely night, a poor Red Panda was turned green. He was thrown into the dungeons and forced to navigate to the safety of his home. Get him home so he can wash that green off!**

**How do you play your game?**

<!-- Press WASD to navigate the maze and move. Reload if you get stuck!-->

QuizzityMC commented:
Oop, sorry! I forgot about the checklist (thought it was just for me). AFAIK, I tick all of those boxes though.

grymmy reviewed (changes requested):
This needs more effort - you can walk through the walls.  Just a maze game is not enough, please add code to introduce new game mechanics.

QuizzityMC commented:
Updated @grymmy ! Thanks for your comment, I have added pushable walls and keys and doors! I've also made the maps more complicated. Hope that's sufficient!

LucasHT22 reviewed (changes requested):
Hey @QuizzityMC! I see inactivity here, if you're struggling with making changes, please let us know:)

QuizzityMC commented:
@LucasHT22 But I finished! It was never re-reviewed, I did update it here 3 weeks ago!!!!! https://github.com/hackclub/sprig/pull/3004/commits/11a625da8c33014b6c9674a9dcd2fa3f022613b4

grymmy reviewed (changes requested):
Please add more complexity to this game to justify a Sprig device grant
